xbmc/rendering/gl/RenderSystemGL.cpp

```diff
@@ -61,6 +61,8 @@ bool CRenderSystemGL::InitRenderSystem()
         m_RenderExtensions += " ";
       }
     }
+
+    KODI::UTILS::GL::VAOsSupported(true);
```

Comment:
Would a better name be `SetVAOsSupported()`?

Reply:
Yep

Reply:
As long as we're on the topic of naming, maybe use our style for public functions, e.g. `GLBindVertexArray`, which means that there are no symbol conflicts and you can drop the `::`.

Reply:
updated. (should I drop the GL from the function name and just leave that in the namespace? ie. `KODI::UTILS::GL::GenVertexArrays`

Reply:
No preference either way